playground/AzureStorageEndToEnd/AzureStorageEndToEnd.ApiService/Program.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Azure.Storage.Blobs;
+
+var builder = WebApplication.CreateBuilder(args);
+
+int insertionRows = builder.Configuration.GetValue<int>("InsertionRows", 1);
```

Comment:
```suggestion
int insertionRows = builder.Configuration.GetValue("InsertionRows", 1);
```

Reply:
Actually I have to remove this. Copypasta.